ui/components/DreamCard.js

```diff
@@ -58,7 +58,7 @@ export default ({ dream }) => {
         <div>
           <h3>{dream.title}</h3>
 
-          <p>{truncate(dream.description, 220)}</p>
+          <p>{dream.summary ? truncate(dream.summary, 200) : ''}</p>
```

Comment:
the backend already limits to 200 characters, so this is not needed

Reply:
however, I think it would be good to have a counter with the remaining available characters shown to the user in the UI

Reply:
Yes, true. Will update.